src/snowflake/snowpark/modin/plugin/docstrings/dataframe.py

```diff
@@ -1185,6 +1185,77 @@ def add():
         Get addition of ``DataFrame`` and `other`, element-wise (binary operator `add`).
         """
 
+    def bfill():
```

Comment:
Why isn't there a docstring for backfill? We have two for ffill and for pad, so either we should add one for backfill, or remove one of either ffill or pad

Reply:
Good point. I added the docstring for backfill similar to pad.